### DISCUSSION TITLE
Bug with Slot Component Backgrounds

### DISCUSSION THREAD

**Discussion by kkenna3** [twilio-labs/paste#3244] [Help: Figma] [answered]
Hi, first off, I'm loving the flexibility of Slots, so thank you! 
In [Sunset] Paste, when replacing slots with my local components, everything switched over fine. With [New] Paste, when I swap the Slot component in the same way, the purple background remains. While it's easy to remove the properties once, it definitely gets tedious with multiple components. Is there a easy fix on the library side, or something designers can do in the meantime?

Here's an example of it happening within a card, but I've experienced it with other components that use slots. I've also tried applying a white background to the custom component but it's still overridden by the purple. 
<img width="1049" alt="Screenshot 2023-05-23 at 1 58 54 PM" src="https://github.com/twilio-labs/paste/assets/102549494/dcb05648-b6a3-496b-93d2-a339c07f8582">

Thanks!

**Comment by TheSisb** [comment]
Hi @kkenna3,

This is a known bug and we have a ticket to fix it in our next sprint. We'll report back when it is fixed.

Thanks for making the bug report!

**Comment by serifluous** [comment, marked as answer]
The fix has now been published. Thanks for catching this!
